ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -1105,6 +1109,10 @@ static void get_key_min_max_sizes(enum pkcs11_key_type key_type,
 
 	mechanism_supported_key_sizes(mechanism, min_key_size,
 				      max_key_size);
+	if (key_size_in_bits) {
+		*min_key_size /= 8;
+		*max_key_size /= 8;
```

Comment:
These should be rounded before conversion to byte, otherwise return min key size for generic secret is 0 byte.
```c
*min_key_size = (*min_key_size + 7) / 8;
*max_key_size = (*max_key_size + 7) / 8;
```

Reply:
Thanks. I will correct it.